src/pages/Login.tsx

```
@@ -0,0 +1,45 @@
+        <S.SocielLoginSentenceArea>
+          <S.SocielLoginSentence>
+            다른 방법으로 로그인 하기
+          </S.SocielLoginSentence>
+        </S.SocielLoginSentenceArea>
+        <S.OAuthIconsArea>
+          {OAUTH_PROVIDERS.map((oauth) => (
+            <S.OAuthIcons
+              onClick={() => handleOAuthLogin(oauth.url, oauth.name)}
+            >
+              <S.IconsImage src={oauth.icon} />
+            </S.OAuthIcons>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)